src/common/k8s.ts

```diff
@@ -162,27 +162,32 @@ type WaitTillAvailableOptions = {
   skipSsl?: boolean
 }
 
-export const waitTillAvailable = async (
-  url: string,
-  options: WaitTillAvailableOptions = { status: 200, skipSsl: false },
-): Promise<void> => {
+export const waitTillAvailable = async (url: string, opts?: WaitTillAvailableOptions): Promise<void> => {
+  const options = { status: 200, skipSsl: false, ...opts }
   const debug = terminal('waitTillAvailable')
   const retryOptions: Options = {
     retries: 50,
     maxTimeout: 30000,
   }
 
-  const rejectUnauthorized = !options.skipSsl
+  const globalSkipSsl = !env().NODE_TLS_REJECT_UNAUTHORIZED
+  let rejectUnauthorized = !globalSkipSsl
+  if (opts!.skipSsl !== undefined) rejectUnauthorized = !options.skipSsl
```

Comment:
if .NODE_TLS_REJECT_UNAUTHORIZED is set to 0, then can we still enforce rejection?

Reply:
I don't assume so...global means node will use that